### PR TITLE
blueprint: Use new Quilt API

### PR DESCRIPTION
**Pull request by kklin**
The Quilt API now deploys containers directly.

### CODE REVIEW

File: node.js [hunk 1]
```diff
@@ -31,25 +32,15 @@ function Node(cfg) {
 
   const env = cfg.env || {};
   const containers = buildContainer(cfg.repo).withEnv(env).replicate(cfg.nWorker);
-  this._app = new Service('app', containers);
+  this.cluster = containers;
```
[the remark this answers is not in the document]
Comment (kklin) [reply node.js]: I'm wary of renaming it now..

File: node.js [hunk 2]
```diff
@@ -31,25 +32,15 @@ function Node(cfg) {
...
-Node.prototype.services = function services() {
-  return [this._app];
+  deployment.deploy(this.cluster);
 };
 
 Node.prototype.port = function port() {
   return this._port;
 };
 
-Node.prototype.allowFrom = function allowFrom(from, port) {
```
[the remark this answers is not in the document]
Comment (kklin) [reply node.js]: Nothing actually used it. I could make it a separate commit if you'd like

[the remark this answers is not in the document]
Comment (kayousterhout) [reply node.js]: nah that's fine I just wanted to double check that was the reason

File: node.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kayousterhout) [reply node.js]: OK

**Pull request merged**